src/Microsoft.OData.Client/ALinq/ResourceBinder.cs

```diff
@@ -797,6 +799,37 @@ internal static Expression ApplyOrdering(MethodCallExpression mce, bool descendi
             return input;
         }
 
+        private static Expression AnalyzeAggregation(MethodCallExpression methodCallExpr, AggregationMethod aggregationMethod)
+        {
+            Debug.Assert(methodCallExpr != null, "methodCallExpr != null");
+            if (methodCallExpr.Arguments.Count != 2)
+            {
+                return methodCallExpr;
+            }
+
+            QueryableResourceExpression input;

```

Comment:
>input [](start = 40, length = 5)

maybe rename it as "output", it's weird to use "input" as output. maybe other name